server/src/telemetry/SentryServerTelemetry.ts

```diff
@@ -13,7 +13,7 @@ const SENTRY_CLOSE_TIMEOUT = 2000;
 export class SentryServerTelemetry implements Telemetry {
   private dsn: string;
   private serverState: ServerState | null;
-  private analytics: Analytics;
```

Comment:
I get the move here, but I suspect we should either:

1. Add analytics as a property of the serverState and call it directly (SentryTelemetry used to be the only place analytics was used, that is no longer the case). Though this likely means reworking `init` and heartbeat.
2. Do this analytics changes on the client